renovate.json

```diff
@@ -0,0 +1,37 @@
+{
+    "branchPrefix": "renovate-",
+    "prHourlyLimit": 0,
+    "lockFileMaintenance": {
+        "enable": true,
+        "schedule": [
+            "before 9am every weekday"
+        ],
+        "automergeType": "branch",
+        "automerge": true
+    },
+    "packageRules": [
+        {
+            "depTypeList": [
+                "devDependencies"
+            ],
+            "rangeStrategy": "replace",
+            "semanticCommitType": "Fix",
+            "commitMessageTopic": "devDependency {{depName}}"
+        },
+        {
+            "depTypeList": [
+                "dependencies"
+            ],
+            "rangeStrategy": "replace",
+            "semanticCommitType": "Feature",
+            "commitMessageTopic": "dependency {{depName}}"
+        },
+        {
+            "schedule": [
```

Comment:
* I think we'll want to specify `timezone` as well.
* Thoughts on not providing a schedule at all for open-source repos?